docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -608,742 +1106,964 @@ Predefined Grid Parameters
    | ``"RRFS_SUBCONUS_3km"`` 
    | ``"WoFS_3km"``
 
-.. note::
+.. note::
+
+   * If ``PREDEF_GRID_NAME`` is set to a valid predefined grid name, the grid generation method, the (native) grid parameters, and the write component grid parameters are set to predefined values for the specified grid, overwriting any settings of these parameters in the user-specified experiment configuration file (``config.yaml``). In addition, if the time step ``DT_ATMOS`` and the computational parameters (``LAYOUT_X``, ``LAYOUT_Y``, and ``BLOCKSIZE``) are not specified in that configuration file, they are also set to predefined values for the specified grid.
+
+   * If ``PREDEF_GRID_NAME`` is set to an empty string, it implies that the user will provide the native grid parameters in the user-specified experiment configuration file (``config.yaml``).  In this case, the grid generation method, the native grid parameters, the write component grid parameters, the main time step (``DT_ATMOS``), and the computational parameters (``LAYOUT_X``, ``LAYOUT_Y``, and ``BLOCKSIZE``) must be set in the configuration file. Otherwise, the values of the parameters in the default experiment configuration file (``config_defaults.yaml``) will be used.
+
+Aerosol Climatology Parameter
+---------------------------------
+
+``USE_MERRA_CLIMO``: (Default: false)
+   Flag that determines whether :term:`MERRA2` aerosol climatology data and lookup tables for optics properties are obtained. Valid values: ``True`` | ``False``
+
+   .. COMMENT: When would it be appropriate to obtain these files?
+
+Surface Climatology Parameter
+---------------------------------
+
+``SFC_CLIMO_FIELDS``: (Default: see below)
+
+   .. code-block:: console 
+
+      [
+         "facsf",
+         "maximum_snow_albedo",
+         "slope_type",
+         "snowfree_albedo",
+         "soil_type",
+         "substrate_temperature",
+         "vegetation_greenness",
+         "vegetation_type"
+      ] 
+   
```

Comment:
@danielabdi-noaa It looks like the SFC_CLIMO_FIELDS are still in config_defaults.yaml post-PR. Should I remove from both config_defaults.yaml and ConfigWorkflow.rst? It does look like you moved them to a new file in your PR, but maybe forgot to delete? Or maybe I'm missing something? I've deleted the other fixed file arrays from the docs though.

Reply:
@gspetro-NOAA Good catch! Please remove it from `config_defaults.yaml`. I don't think it makes a difference since it is a constant array, but I will make some tests anyway.